src/rsa/signing.rs

```diff
@@ -176,10 +213,18 @@ impl RSAKeyPair {
                     return Err(error::Unspecified);
                 }
 
+                // [NIST SP-800-56B rev. 1]] 6.4.1.4.3 - Step 6
+                //
+                // TODO 6.a: d < LCM(p – 1, q – 1)
+                // TODO 6.b: 1 == (d · e) mod LCM(p – 1, q – 1)
```

Comment:
> d is fully validated neither at construction nor during signing.

Your suggested wording sounds good to me, except I don't like starting this sentence with "d is fully validated neither" since it could easily be misread as "d is fully validated". I'd rather start it saying "d is never fully validated, neither" or some such.